x-pack/agentbeat/magefile.go

```diff
@@ -205,7 +211,13 @@ func GoIntegTest(ctx context.Context) error {
 	mg.Deps(BuildSystemTestBinary)
 	args := devtools.DefaultGoTestIntegrationFromHostArgs()
 	args.Tags = append(args.Tags, "agentbeat")
-	args.Packages = append(args.Packages, "../auditbeat/...", "../filebeat/...", "../heartbeat/...", "../osquerybeat/...", "../packetbeat/...")
+	for _, beat := range getIncludedBeats() {
+		// original code never included metricbeat integration tests, see
+		// https://github.com/elastic/beats/blob/873743b78a16c16eb74fd0e3ac538769e63acfab/x-pack/agentbeat/magefile.go#L208
+		if beat != "metricbeat" {
```

Comment:
Was the previous exclusion of metricbeat intentional? It would be nice to have a comment about why it's skipped instead of just referencing previous behavior

Reply:
As far as I can tell the integration tests for metricbeat don't work because require some binary that is not built when compiling agentbeat.
 You can see the failures in a previous run of this PR, I discovered it this morning while comparing the diffs trying to figure out why the tests were broken.

@blakerouse maybe can offer more context about it.

Reply:
@faec I modified the comment about the why this is excluded (I don't know if this is the same as the original reason). Please have another look.